demos/link-prediction/random-walks/ctdne-link-prediction.ipynb

```diff
@@ -0,0 +1,680 @@
+{
```

Comment:
<p>I would suggest to split this into three steps.</p><ol><li>Hyperparameters setting.</li><li>Temporal Random Walks</li><li>Static Random Walks</li></ol><p>Too much going on in one place.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1056/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
I've split this up now 👍